### DISCUSSION TITLE
사파리 데스크톱 사이트 요청

### DISCUSSION THREAD

**Discussion by kangsiwoo** [Bamdoliro/marururu#400] [General]
아이폰에 있는 사파리에서 데크크톱 사이트를 요청하면 사이트에 진입이 가능합니다.
그리고 그 사이트의 가독성이 상당히 깨집니다.
[가로](https://lh3.googleusercontent.com/fife/AKsag4NcouS07iDGIrrPJwI68SWQuoiH1gcxkjE6nH1ksHOeZmwwNWagaKlXHaytd7OJQieX9jGMZfAGuShpA8alzVh6Z3G1jv2PfJ7zLbBih7rATjuZy6_hSyBBYlFdmLWhaucd5eSRllUqRARrJ6AHRBChpDKuxmWmA1ay-6XbtQMYEvYPb4ppzH7Ukk-Rf-VnYlGHXWe51DzCJBvziU22Tfd4qaqwBBBufQs6aAHy6JyBiznHUL2Otjda82Zo14HYf3JWSHHc9x1D1-g8clhx6lCHXkvsu19Fy5SxgBJFdcWtP40sLsf62PlXO4ILAzsb6X7q-fBmtrZBFA0EmL-i7U7SrlJkR2LH8R5B-ML5fLVn5M3HLozb6-CVVDdt89HKxNg3e-8EXYsmQoZA6bvRr-RcND-CHXBTdMWsjhDIX5_tmE8XJuIl9ThUKXR60SYpIc2aesvzbHTg-PiqQutgqpgLfnXq2ePkwyvmKMNDL87Hj17fTA9Sfyw3w7vUVc6euG_SyNM1f_nxwLiAvnMIEN6CmmPoxRrTMkkyQiFxy9_-zxkqxJ1hvFnFTeSymlp1udW-1RM8-_5vel9tX9P5zwbEzgoYNYe2mv7X_zSrbPBtOGm0gbQ-yL-5CSKjoT8TH5LUJWGnsrr2t3jQrBIyRs1p-LqxSnSb55Nj0Qe_Zr5k8N99flFvmRVanUjQY-FzgkQ1lR3FhztBvPwpo_EOOWFUguFIg4F85pU3AItVDnzjFNvS_0eBdslok5p2LxgLSVfbVT0XE5URoWpNkTCjvd6rJAOxtFAeWH2nWGRr4zuO7CQ4_BfYbqiTFBz0ZU2DR1LLZc-zSrjHyivJlSP_eCoP3mhSyd5-gN3bwyQdYFPNORHlkYOuft8VkF6EVWVbxREYhBuEof_8fNP8_pRNbHdZKwvb8lnBdudI2HuWvIUhkWc3gJDy7BCVU2dlWG7EugC-nnl5nfNL-W4TXpuZwNW_2ALP_7qUt60Dbt281fJbr452seJt4YPd92zf4y7wbG-Em_PC4oj1yogn5e8BUY2oY62iRWrVop06YwZi-5fBgGwqHZQCX8hlqrkTG-uJOWdInYhMFj9xuDHgfkQ4I-raVhWoMpQHeRbmH-8MmeibjC000ydGhoig8zXFHuNJPPhkMyydqQ1GxfsuI9y95VFT43cE3Q9EShPv7fbae3msILHlawvqaNY5IVWu43K887sDRGI_jfs3f-ifqqFnlHBDphzFz5rxvXRxWP4nwK50zTtgyqUD4XkhHFLb7jo3jX2RHDHEZRXo_kQIdZNBUMzhFmD5SUN8v5kbyHFNQ-fPjf9_-xw0TJ0g49PM-pDn9f-G2hGLIhB0kSsRqBa7tehR4XzUzT-JprjMB55OH0kBznbO0UVm3Lb6LYaUzu8i5YRZK-2udnHrlrgG1fCURuYd4FyGjUi4oSvEngp9O2NQElCTpvqyxuqMh3jlfEiPYhml1NG9_afnYYvaUJLHpWMYeMwrJBh1OJo_7_j3ZFRzzfGW6sTYDf_NeEYUGtOPrmRWzstnk6FQzblSiV0zDVO6mIVlCkwGqkj-IhmBySQjAXex2xu_YZPTnjLg9Vkimdq0G9_zrmDgdCw4qsszH-d1TvLClnIqpM3-ii3szjozzjuT4k7mbAhAXkYzEaJUrifRSkLMaXl_-HB3QDMumkzWizWRcMDtx98=w1920-h963) | [세로](https://lh3.googleusercontent.com/fife/AKsag4MidsZ_B2UGa7DMwuyrq5GLoWlVfez3g1ll0eGL24W28AfZ0KBr7W2kKaciu0PQruK8bWKYxwyRFs_glDyT3My8Hrh1SihU9_23a8m4-xgOR5klHZg90q5EHX0s6h7gejFCJEiUsmMZ8s8sWyDnmlNPKlbo0pxblvTABKUA0vaimfEg1M9Xr2FiFL1XF0761iQCmgyAbHO_QCJj8cMflOqjqRo-heTr2FXksYmRccFz3CPNOyCh0MQf6r4aWwYQl6hpGYB_kwBpgDEIHRbeonBa8rLdndWs9h2bYXMRk9gNy2qDdW0RQ3QFm7OMRQMx0BJJu9XX8k8U3UjEP7uROGdQKIZuxD04rNdTVG4T2aEBTzm13e-TsXFoEJ3bIyXbPP_yyiZ0Bq0ImBcPx8xbKJ9YYwOLe3zUjc7AtHsBafWRmotINV4d4GKimKAqhR8iUGk8I6kUz9tRrxy9x7cQ0VN4mbYdYL4R6zPoNoM3vpzbe-Q5nths3sXPaJ6clZ1Ybln76wWNGBrYgP3iwhg4wyVwusNKfwT8fbIqFJ6vDf-bW1gVa3ED5IpAa17eFnOi32qfp-JGuhTJZa4iGQAZutk9LM_cYEcWCrLAFjhWSlz_r0VxlW-boVoIRLx2jqqHWpBMGPZP1CqnU7sJzyamVG7bUcGckgGpiOk8n6IMsA7ChL3oH28K8nvFcMqer4F5DgETECIDgBnZCvol-NLuQOhgZYXyVc3avo5roYcb-aodUBEJY1HeWJIPowKfDTlCEgxX27SWVYCimUtoZ8bJuPCdAYn7NCsFheWx7afCu0DOcy8mZPqiLs0vCdbRxdI5pxufjHE1E1QiADJbGr57HZcsDM3HpYlm7GbMy2E0i4nO_oO23NdRQz14L18e5wavUgksc5nsajNRMn_0vufyGbPwSHklOZK37T01XrHxEb9iXjz9nsT8WDp-bbGaQYRy13y1QW_JWSpXDjUgk3fTNouhOTbDdobrfSxwPyKqAmuNZ5p0ehy8tY3F9t6KHmAprEKDDgMjUxW3Hbv2_8J_BGT4rqNEfUUyd9g7JV0abhVRieWjDoc1dQTnj7cCE7o5ne-LLHA7rp37Fr3DTfTb8JiboTUd4MgsTpm6nfTJ3omNKvrjoHReoW7HbMDi9SwfouW-NwMfvyWMrNWXFsCeXP2bGQiXKXLP9BOy6ECI8xzKAir4N783JYU2C9LLSlm-WbPdfYpvVmvHEEUi8ktjUT9s-lTp_SAI4l-gF-20IRN9SXJFK-upYHGqPY_MzufdKBYsR3em57XbLDlbxLjv9-48bFmYtMLyNwFbbwr4MfA7b-80HRrnl9hkVun_S-OE3Q_kbPDsSkmgHUdhOQCNkvzXJc6hfmSUgw1KqYZKR7HerIizF1PUZ-DqE-ZrfKoKkGzEsq8e5VQizxVayM-O6O4fdG5dXh-eF6FgnK-ZDQtwkZi6xFktBMdeutQaVdJMp1ynUwgThPpnJs5FwNL9dbkppnfo8-adSpEoEGuNRWVsuxQTKUxPm_yiFq9O2f51D1Zcm36aUkJvrmrqnZisuyg0NRZUHZy1iRK1leTT7l4DznlcIWMJJSQ4nFsgJKTxpOTucuB1oaZiFHuWi2p_pOR1hO_AAzB9B71PyF1D1aofX9pwnp26FhCofHseZxMLdC-AIReJ7SuLILUhjw4myIL70u8i_6uS5Nw=w1920-h963)
이런 문제는 상관 없나요?

**Comment by SEOKKAMONI** [comment]
헉 사파리에서는 진입이 가능하다는 사실을 모르고 있었습니다 고마워요!!!!!
